src/Umbraco.Web.Common/Localization/UmbracoRequestLocalizationOptions.cs

```diff
@@ -11,18 +11,22 @@ namespace Umbraco.Cms.Web.Common.Localization
     /// </summary>
     public class UmbracoRequestLocalizationOptions : IConfigureOptions<RequestLocalizationOptions>
     {
-        private readonly IOptions<GlobalSettings> _globalSettings;
+        private GlobalSettings _globalSettings;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="UmbracoRequestLocalizationOptions"/> class.
         /// </summary>
-        public UmbracoRequestLocalizationOptions(IOptions<GlobalSettings> globalSettings) => _globalSettings = globalSettings;
+        public UmbracoRequestLocalizationOptions(IOptionsMonitor<GlobalSettings> globalSettings)
```

Comment:
I think this is transient too, if you navigate to implementation of `ConfigureOptions` in `AddWebComponents` you can see that it gets added as transient